src/test/resources/sql/scheduledPrisonerReturnFromCourt_init.sql

```diff
@@ -0,0 +1,139 @@
+        'URUA', 'G6942UN', 2554472, 'INST', 'OMS_OWNER',
+        TO_DATE('2017-06-26', 'YYYY-MM-DD'), 'CN', null, null, null, null, 'W1',
+        null, null, null, null, null, 'MR', null, 'QQM25S', 'A',
+        TO_DATE('2017-02-17', 'YYYY-MM-DD'), '1234',
+        TO_DATE('2017-06-26', 'YYYY-MM-DD'), 'OMS_OWNER', 'MERGE', 'pgrant',
+        '10.102.1.69', 'unknown', null);
+
+INSERT INTO AGENCY_LOCATIONS (AGY_LOC_ID, DESCRIPTION, AGENCY_LOCATION_TYPE, ACTIVE_FLAG)
+VALUES ('NMI', 'NOTTINGHAM (HMP)', 'INST', 'Y'),('DRBYCC', 'Derby Crown Court', 'CRT', 'Y');
+
+
+
+INSERT INTO AGENCY_INTERNAL_LOCATIONS (INTERNAL_LOCATION_ID, INTERNAL_LOCATION_CODE, AGY_LOC_ID,
+                                                 INTERNAL_LOCATION_TYPE, DESCRIPTION, SECURITY_LEVEL_CODE, CAPACITY,
+                                                 CREATE_USER_ID, PARENT_INTERNAL_LOCATION_ID, ACTIVE_FLAG, LIST_SEQ,
+                                                 CREATE_DATETIME, MODIFY_DATETIME, MODIFY_USER_ID, CNA_NO,
+                                                 CERTIFIED_FLAG, DEACTIVATE_DATE, REACTIVATE_DATE,
+                                                 DEACTIVATE_REASON_CODE, COMMENT_TEXT, USER_DESC, ACA_CAP_RATING,
+                                                 UNIT_TYPE, OPERATION_CAPACITY, NO_OF_OCCUPANT, TRACKING_FLAG,
+                                                 AUDIT_TIMESTAMP, AUDIT_USER_ID, AUDIT_MODULE_NAME,
+                                                 AUDIT_CLIENT_USER_ID, AUDIT_CLIENT_IP_ADDRESS,
+                                                 AUDIT_CLIENT_WORKSTATION_NAME, AUDIT_ADDITIONAL_INFO)
+VALUES (106221, 'F', 'NMI', 'WING', 'NMI-F', null, 116, 'OMS_OWNER', null, 'Y', 10,
+        TO_DATE('2009-12-10', 'YYYY-MM-DD'),
+        TO_DATE('2021-11-19', 'YYYY-MM-DD'), 'API_PROXY_USER', 78, 'Y', null,
+        null, null, null, null, null, 'NA', 102, 93, 'N',
+        TO_DATE('2021-11-19', 'YYYY-MM-DD'), 'GCHEEMA_GEN', 'PRISON_API',
+        'GCHEEMA_GEN', '35.177.252.54', 'API', 'PUT /api/bookings/1182239/living-unit/NMI-F-1-001'),
+       (106249, '2', 'NMI', 'LAND', 'NMI-F-2', null, 40, 'OMS_OWNER', 106221, 'Y', 2,
+        TO_DATE('2009-12-10', 'YYYY-MM-DD'),
+        TO_DATE('2017-05-09', 'YYYY-MM-DD'), 'OMS_OWNER', 26, 'Y', null, null,
+        null, null, null, null, 'NA', null, 34, 'N',
+        TO_DATE('2017-05-09', 'YYYY-MM-DD'), 'OMS_OWNER',
+        'sqlplus@DAPDL0001 (TNS V1-V3)', 'oracle', null, 'pts/6', null),
+       (106258, '009', 'NMI', 'CELL', 'NMI-F-2-009', null, 2, 'OMS_OWNER', 106249, 'Y', 9,
+        TO_DATE('2009-12-10', 'YYYY-MM-DD'),
+        TO_DATE('2017-05-09', 'YYYY-MM-DD'), 'OMS_OWNER', 1, 'Y', null, null,
+        null, null, null, null, 'NA', null, 2, 'N',
+        TO_DATE('2017-05-09', 'YYYY-MM-DD'), 'OMS_OWNER',
+        'sqlplus@DAPDL0001 (TNS V1-V3)', 'oracle', null, 'pts/6', null);
+
+INSERT INTO MOVEMENT_REASONS (MOVEMENT_TYPE, MOVEMENT_REASON_CODE, DESCRIPTION, OPEN_CONTACT_FLAG,
```

Comment:
If this is a valid MOVEMENT_REASON then I think it should be put in a new flyway script and inserted once as static data.

Reply:
done